e2etest/bgp_test.go

```diff
@@ -191,15 +198,21 @@ var _ = ginkgo.Describe("BGP", func() {
 			for _, c := range frrContainers {
 				validateService(cs, svc, epNodes, c)
 			}
-		})
+		}
 
-		ginkgo.It("would check that FRR is running in the speaker", func() {
+		if setProtocoltest == "CheckSpeakerFRRPodRunning" {
 			for _, c := range frrContainers {
 				paired := frrIsPairedOnPods(cs, c)
 				framework.ExpectEqual(paired, true)
 			}
-		})
-	})
+		}
+	},
+		table.Entry("IPV4 - FRR ExternalTrafficPolicyCluster", "ipv4", "ExternalTrafficPolicyCluster"),
+		table.Entry("IPV4 - FRR ExternalTrafficPolicyLocal", "ipv4", "ExternalTrafficPolicyLocal"),
+		table.Entry("IPV4 - FRR Check speaker POD is running", "ipv4", "CheckSpeakerFRRPodRunning"),
```

Comment:
Sorry, I was probably not clear in my previous comment. What I meant is to add FRR only in the name of the test that will work only in the FRR implementation (which is the one with `CheckSpeakerFRRPodRunning` where we check if FRR is running inside the pod).

All the others must not contain FRR or they'll be skipped even in the native lane, which is skipping everything with FRR in the test name.
Also, I'd rename `Check speaker POD is running` to `FRR running in the speaker POD`

Reply:
`pairExternalFRRWithNodes` is invoked on all tests and will fail on native run

Reply:
And that's why its entry (and only its entry) must contain FRR in the name. All the others should not otherwise the native will skip them (even if it makes sense to run it)